### ISSUE TITLE
choropleth + world-population example combined

### ISSUE THREAD

**Issue by gomedya** [vasturiano/globe.gl#83]
hi , is it possible to set different heights of individual polygons in choropleth example and keep them fixed like world-pop example?

sort of like Elevated Polygons but with walls/sides going all the way down to globe.

Thanks.

**Comment by vasturiano** [comment]
@gomedya there is the `.polygonAltitude` method available exactly for that purpose.

**Comment by gomedya** [comment]
hi , when you hover in choropleth example , countries go up and down. i need to specify each country a different altitude and not go up and down. it will look like a bar chart , countries with different heights.

**Comment by vasturiano** [comment]
Yes, that should all be easily doable using `polygonAltitude`. You can see a similar use of that in the [Elevated Polygons example](https://globe.gl/example/countries-population/), particularly this line:
https://github.com/vasturiano/globe.gl/blob/999c2c139327dddeffeb5478d7eafd26ec3026c1/example/countries-population/index.html#L32

**Comment by gomedya** [comment]
that did work like charm. is there an easy way to map the click to country name?

**Comment by vasturiano** [comment]
The `onPolygonClick` event includes the polygon object that was clicked. You should be able to extract from that any meta data that's available in the original data set.

**Comment by gomedya** [comment]
that was great help that came with great library. if I end up using it , i'll be more than happy to pay it back. thank you.
